lld/ELF/Driver.cpp

```diff
@@ -3062,6 +3062,31 @@ void LinkerDriver::link(opt::InputArgList &args) {
     script->addOrphanSections();
   }
 
+  // Explicitly demote symbols which didn't get placed. If we don't, any
```

Comment:
"Explicitly" is redundant.

Suggest: Demote symbols defined relative to input sections that are discarded by /DISCARD/ so that relocations referencing them will get reported.